homeassistant/components/sensor/strings.json

```diff
@@ -23,7 +23,7 @@
       "is_illuminance": "Current {entity_name} illuminance",
       "is_irradiance": "Current {entity_name} irradiance",
       "is_moisture": "Current {entity_name} moisture",
-      "is_monetary": "Current {entity_name} money",
+      "is_monetary": "Current {entity_name} balance",
```

Comment:
This device class doesn't represent a total, thus cannot be a balance.

Reply:
Well, I mentioned it at the very top of my PR, this is what the 'monetary' entity class is named:

https://github.com/home-assistant/core/blob/d8549409f79aeebe6ae8650849deb219b4cf6461/homeassistant/components/sensor/strings.json#L210-L211

It is the current balance on some (virtual) account.

And the trigger fires when that balance is above or below a given value:

    When  entity_name} balance changes  (above / below xxx)

And the condition checks if the balance is above, below or between these two limits.

    If  current {entity_name} balance   is  above xxx (etc.)

What else would you use in such a trigger or condition?

Reply:
Now if that is called "Balance", a trigger "when xyz money changes" and a condition "if xyz money is above / below …" does not make sense.

But a trigger "when xyz balance changes" and a condition "if xyz balance is above / below …" sound like the logical way to work with such a device class.

Reply:
> This device class doesn't represent a total, thus cannot be a balance.

https://github.com/home-assistant/core/blob/c8480627ca40d0d52d1f4f590fb7b99668d35eb5/homeassistant/components/sensor/const.py#L633-L634

Reply:
Check 👍 

Sorry for the confusion there 👍